cmd/podman/common/create_opts.go

```diff
@@ -25,7 +28,7 @@ type ContainerCLIOpts struct {
 	CPUSetMems        string
 	Detach            bool
 	DetachKeys        string
-	Device            []string
+	Devices           []specs.LinuxDevice
```

Comment:
You cannot change the variable name or type in this structure as they need to match the createflags. cobra is not aware of a LinuxDevice.

Reply:
I am not using this value in Cobra.

Reply:
I could just remove it, and not use it to pass this to specgen  But then I need to add a function.